ui/app/src/components/TokenManagement/TokenManagement.tsx

```diff
@@ -0,0 +1,452 @@
+              <ActionsBar
+                onOptionClicked={onOptionClicked}
+                options={options}
+                isIndeterminate={checkedCount > 0 && checkedCount < tokens.length}
+                isChecked={checkedCount === tokens.length}
+                toggleSelectAll={() => onToggleSelectAll(checkedCount !== tokens.length)}
+                classes={{
+                  root: classes.actionsBar
+                }}
+              />
+            )}
+            <Table className={classes.table}>
+              <TableHead>
+                <TableRow>
+                  <TableCell padding="checkbox">
+                    <Checkbox
+                      disabled={checkedCount > 0}
+                      checked={false}
+                      color="primary"
+                      onClick={() => {
+                        onToggleSelectAll(true);
+                      }}
+                    />
+                  </TableCell>
+                  <TableCell align="left" padding="none">
+                    <Typography variant="subtitle2">
+                      <FormattedMessage id="words.status" defaultMessage="Status" />
+                    </Typography>
+                  </TableCell>
+                  <StyledTableCell align="left">
+                    <Typography variant="subtitle2">
+                      <FormattedMessage id="words.label" defaultMessage="Label" />
+                    </Typography>
+                  </StyledTableCell>
+                  <StyledTableCell align="left">
+                    <Typography variant="subtitle2">
+                      <FormattedMessage id="words.expiration" defaultMessage="Expiration" />
+                    </Typography>
+                  </StyledTableCell>
+                  <StyledTableCell align="left">
+                    <Typography variant="subtitle2">
+                      <FormattedMessage id="words.created" defaultMessage="Created" />
+                    </Typography>
+                  </StyledTableCell>
+                  <TableCell align="center" className={classes.actions} />
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {tokens.map((token) => (
+                  <TableRow key={token.id}>
+                    <TableCell padding="checkbox">
+                      <Checkbox
+                        checked={!!checkedLookup[token.id]}
+                        color="primary"
+                        onChange={(event, checked) => {
+                          onItemChecked(token.id, checked);
+                        }}
+                      />
+                    </TableCell>
+                    <TableCell component="th" id={token.id.toString()} scope="row" padding="none">
+                      <Chip
+                        label={
+                          moment(token.expiresAt) < moment() ? (
+                            <FormattedMessage id="words.expired" defaultMessage="Expired" />
```

Comment:
let’s store the moment/date on a car to avoid the repeated call 
Do you really need to use moment? Looks like simple dates would do